[(0, "\n\nimport sys\nsys.argv = ['/home/jchop/py_dna/130924_dna_plots.py']\n", '', '', ()), (1, '#!/usr/bin/python\n', '', '', ()), (2, 'import matplotlib               \n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/__init__.py", line 134, in <module>\n    from matplotlib.rcsetup import (defaultParams,\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/rcsetup.py", line 18, in <module>\n    from matplotlib.fontconfig_pattern import parse_fontconfig_pattern\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/fontconfig_pattern.py", line 23, in <module>\n    from matplotlib.pyparsing import Literal, ZeroOrMore, \\\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 3207, in <module>\n    _escapedHexChar = Combine( Suppress(_bslash + "0x") + Word(hexnums) ).setParseAction(lambda s,l,t:unichr(int(t[0],16)))\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 2923, in __init__\n    self.leaveWhitespace()\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 2565, in leaveWhitespace\n    self.expr = self.expr.copy()\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 681, in copy\n    cpy = copy.copy( self )\n  File "/usr/lib/python2.7/copy.py", line 88, in copy\n    rv = reductor(2)\nTypeError: __call__() takes at most 5 arguments (6 given)\n\n', ()), (3, 'import numpy as np                  \n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/python2.7/dist-packages/numpy/__init__.py", line 137, in <module>\n    import add_newdocs\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/python2.7/dist-packages/numpy/add_newdocs.py", line 9, in <module>\n    from numpy.lib import add_newdoc\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/python2.7/dist-packages/numpy/lib/__init__.py", line 4, in <module>\n    from type_check import *\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/python2.7/dist-packages/numpy/lib/type_check.py", line 8, in <module>\n    import numpy.core.numeric as _nx\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/python2.7/dist-packages/numpy/core/__init__.py", line 6, in <module>\n    import umath\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\nImportError: numpy.core.multiarray failed to import\n\n', ()), (4, 'from pylab import *\n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\n  File "/usr/bin/pyreport", line 584, in __call__\n    return self.pylab_import(name, globals, locals, fromlist)\n  File "/usr/bin/pyreport", line 591, in pylab_import\n    matplotlib = self.original_import("matplotlib")\n  File "/usr/lib/pymodules/python2.7/matplotlib/__init__.py", line 134, in <module>\n    from matplotlib.rcsetup import (defaultParams,\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/rcsetup.py", line 18, in <module>\n    from matplotlib.fontconfig_pattern import parse_fontconfig_pattern\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/fontconfig_pattern.py", line 23, in <module>\n    from matplotlib.pyparsing import Literal, ZeroOrMore, \\\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 3207, in <module>\n    _escapedHexChar = Combine( Suppress(_bslash + "0x") + Word(hexnums) ).setParseAction(lambda s,l,t:unichr(int(t[0],16)))\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 2923, in __init__\n    self.leaveWhitespace()\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 2565, in leaveWhitespace\n    self.expr = self.expr.copy()\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 681, in copy\n    cpy = copy.copy( self )\n  File "/usr/lib/python2.7/copy.py", line 88, in copy\n    rv = reductor(2)\nTypeError: __call__() takes at most 5 arguments (6 given)\n\n', ()), (5, 'from matplotlib import pyplot as pl\n#from matplotlib import axis as ax                                  \n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/__init__.py", line 134, in <module>\n    from matplotlib.rcsetup import (defaultParams,\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/rcsetup.py", line 18, in <module>\n    from matplotlib.fontconfig_pattern import parse_fontconfig_pattern\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/fontconfig_pattern.py", line 23, in <module>\n    from matplotlib.pyparsing import Literal, ZeroOrMore, \\\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 3207, in <module>\n    _escapedHexChar = Combine( Suppress(_bslash + "0x") + Word(hexnums) ).setParseAction(lambda s,l,t:unichr(int(t[0],16)))\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 2923, in __init__\n    self.leaveWhitespace()\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 2565, in leaveWhitespace\n    self.expr = self.expr.copy()\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 681, in copy\n    cpy = copy.copy( self )\n  File "/usr/lib/python2.7/copy.py", line 88, in copy\n    rv = reductor(2)\nTypeError: __call__() takes at most 5 arguments (6 given)\n\n', ()), (7, 'from matplotlib.ticker import MultipleLocator#, FormatStrFormatter\n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/__init__.py", line 134, in <module>\n    from matplotlib.rcsetup import (defaultParams,\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/rcsetup.py", line 18, in <module>\n    from matplotlib.fontconfig_pattern import parse_fontconfig_pattern\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/fontconfig_pattern.py", line 23, in <module>\n    from matplotlib.pyparsing import Literal, ZeroOrMore, \\\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 3207, in <module>\n    _escapedHexChar = Combine( Suppress(_bslash + "0x") + Word(hexnums) ).setParseAction(lambda s,l,t:unichr(int(t[0],16)))\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 2923, in __init__\n    self.leaveWhitespace()\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 2565, in leaveWhitespace\n    self.expr = self.expr.copy()\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 681, in copy\n    cpy = copy.copy( self )\n  File "/usr/lib/python2.7/copy.py", line 88, in copy\n    rv = reductor(2)\nTypeError: __call__() takes at most 5 arguments (6 given)\n\n', ()), (8, "from matplotlib.backends.backend_pdf import PdfPages\n#pp = PdfPages('plots/130729_Hopkins_shifted_df_DF.pdf')                                                        \n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/__init__.py", line 134, in <module>\n    from matplotlib.rcsetup import (defaultParams,\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/rcsetup.py", line 18, in <module>\n    from matplotlib.fontconfig_pattern import parse_fontconfig_pattern\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/fontconfig_pattern.py", line 23, in <module>\n    from matplotlib.pyparsing import Literal, ZeroOrMore, \\\n  File "/usr/bin/pyreport", line 587, in __call__\n    return self.original_import(name, globals, locals, fromlist)\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 3207, in <module>\n    _escapedHexChar = Combine( Suppress(_bslash + "0x") + Word(hexnums) ).setParseAction(lambda s,l,t:unichr(int(t[0],16)))\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 2923, in __init__\n    self.leaveWhitespace()\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 2565, in leaveWhitespace\n    self.expr = self.expr.copy()\n  File "/usr/lib/pymodules/python2.7/matplotlib/pyparsing.py", line 681, in copy\n    cpy = copy.copy( self )\n  File "/usr/lib/python2.7/copy.py", line 88, in copy\n    rv = reductor(2)\nTypeError: __call__() takes at most 5 arguments (6 given)\n\n', ()), (10, "\nx_t ,y_t = np.loadtxt('data/CG10e2t.csv',delimiter=',',unpack=True, usecols = [0,1])\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 4, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (12, "x_x ,y_x = np.loadtxt('data/CG10e2x.csv',delimiter=',',unpack=True, usecols = [0,1])\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (13, "x_y ,y_y = np.loadtxt('data/CG10e2y.csv',delimiter=',',unpack=True, usecols = [0,1])\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (14, "x_z ,y_z = np.loadtxt('data/CG10e2z.csv',delimiter=',',unpack=True, usecols = [0,1])\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (15, '\nz_t = np.ma.polyfit(x_t, y_t, 3)#deg, rcond=None, full=False, w=None, cov=False)\n#z_t = np.polynomial.polynomial.polyfit(x_t, y_t,10)#deg, rcond=None, full=False, w=None, cov=False)                                                                                                    \n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 4, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (18, 'z_t_fit = np.poly1d(z_t)\n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (19, 'x_fit = np.linspace(0,50,50)\n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (20, '\nz_z = np.ma.polyfit(x_z, y_z, 5)#deg, rcond=None, full=False, w=None, cov=False)\n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 4, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (22, 'z_z_fit = np.poly1d(z_z)\n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (23, 'x_fit = np.linspace(0,50,25)\n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (24, 'y_interp = np.interp(x_fit, x_t,y_t)\n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'np\' is not defined\n\n', ()), (25, '\npl.figure()\n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 4, in <module>\nNameError: name \'pl\' is not defined\n\n', ()), (27, "pl.plot(x_t,y_t, label = 'total')\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'pl\' is not defined\n\n', ()), (28, "pl.plot(x_x,y_x, label = r'$\\hat{x}$')\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'pl\' is not defined\n\n', ()), (29, "pl.plot(x_y,y_y, label = r'$\\hat{y}$')\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'pl\' is not defined\n\n', ()), (30, "pl.plot(x_z,y_z, label = r'$\\hat{z}$')\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'pl\' is not defined\n\n', ()), (31, "pl.plot(x_fit,z_t_fit(x_fit), label = r'$t\\, fit$')\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'pl\' is not defined\n\n', ()), (32, "pl.plot(x_fit,y_interp, label = r'$t\\, interp$')\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'pl\' is not defined\n\n', ()), (33, "pl.xlabel(r'$\\hbar\\omega\\,\\,\\,[eV]$', size = 24)\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'pl\' is not defined\n\n', ()), (34, "pl.ylabel(r'$\\varepsilon^{\\prime\\prime}(\\omega)$', size = 24)\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'pl\' is not defined\n\n', ()), (35, 'pl.legend()\n', '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'pl\' is not defined\n\n', ()), (36, "pl.savefig('plots/DNA_spectr_tot_x_y_z.pdf')\n", '', 'Traceback (most recent call last):\n  File "/usr/bin/pyreport", line 477, in __call__\n    exec block_text in self.namespace\n  File "<string>", line 3, in <module>\nNameError: name \'pl\' is not defined\n\n', ())]